front-end/src/utils/utils.js

```diff
@@ -0,0 +1,13 @@
+import moment from 'moment';
+
+const getDate = () => (moment());
```

Comment:
have a date/time service to do this instead of isolated helpers. Group them!

Reply:
can you explain it?

Reply:
modulize this in a file under utils, for example, about time/date. 
/utils/date.js

And in it you can just have

DateHelpers.getDate() , DateHelpers.getTimeFromNow() etc etc

Reply:
yeah, makes sense, thanks!